std/node/assert.ts

```diff
@@ -1,25 +1,45 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+export { AssertionError } from "./assertion_error.ts";
 import {
-  assertEquals,
-  assertMatch,
-  assertNotEquals,
-  assertNotStrictEquals,
-  assertStrictEquals,
-  assertThrows,
+  assertEquals as deepStrictEqual,
+  AssertionError,
+  assertMatch as match,
+  assertNotEquals as notDeepStrictEqual,
+  assertNotStrictEquals as notStrictEqual,
+  assertStrictEquals as strictEqual,
+  assertThrows as throws,
+  fail,
 } from "../testing/asserts.ts";
 
-export { AssertionError } from "./assertion_error.ts";
+function assert(expr: unknown, msg = ""): asserts expr {
+  if (!expr) {
+    throw new AssertionError(msg);
+  }
+}
+const ok = assert;
+export default assert;
 
-export {
-  assert,
-  assert as default,
-  assert as ok,
+// @deprecated use `import assert from 'assert'` as the "assert" export is not a Node.js export
+export { assert };
```

Comment:
@guybedford `std` is still unstable. Feel free to just remove outright, rather than deprecating.

Reply:
Sure, if you're happy to do that aligning now would make sense definitely.